app/uk/gov/hmrc/agentclientrelationshipsfrontend/views/client/ConsentInformationPage.scala.html

```diff
@@ -23,19 +23,34 @@
 
 @this(
         layout: Layout,
+        govukDetails: GovukDetails,
         govukButton: GovukButton
 )
 
-@(journey: ClientJourney)(implicit request: ClientJourneyRequest[?], messages: Messages, appConfig: AppConfig)
+@(journey: ClientJourney, agentRoleKey: String)(implicit request: ClientJourneyRequest[?], messages: Messages, appConfig: AppConfig)
 
 @key = @{"consentInformation"}
 @serviceKey = @{journey.getServiceKey}
 @agentName = @{journey.getAgentName}
-@pageTitle = @{messages(s"$key.header")}
+@pageTitle = @{messages(s"$key.header", agentName, agentRole)}
 @serviceName = @{messages("service.name.clients")}
-@agentRole = @{if appConfig.emaEnabled then messages(s"agentRole.$serviceKey") else messages("agentRole.noEMA")}
+@agentRole = @{if appConfig.emaEnabled then messages(s"agentRoleFor.$serviceKey") else messages("agentRole.noEMA")}
 @indefiniteAgentRole = @{if appConfig.emaEnabled then messages(s"indefiniteAgentRole.$serviceKey") else messages("indefiniteAgentRole.noEMA")}
 @taxServiceName = @{messages(serviceKey)}
+@removeContent = {<p class="govuk-body">@Html(messages(s"$key.section3.p1", routes.ManageYourTaxAgentsController.show.url))</p>}
+@extAgents= @{journey.existingMainAgent}
+@warningHtml= @{(journey.existingMainAgent, agentRoleKey) match {
+    case (Some(exAgent), "suppAgent") if exAgent.sameAgent =>
+        val text = messages(s"$key.warning.mainToSupp", agentName)
+        Html(s"""<p class="govuk-inset-text">$text</p>""")
+
+    case (Some(exAgent), role) if role != "suppAgent" =>
+        val text = messages(s"$key.warning.mainAgent", agentName, exAgent.agencyName)
```

Comment:
the use of `warning.mainAgent` in the message key will not allow us to talk about other service replacements (this content is for ALL services and currently handles non-ITSA use cases) we should use `warning.$agentRole` instead to ensure we capture the use case of `agent`